src/fn_call.rs

```diff
@@ -54,11 +58,12 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a + 'mir>: crate::MiriEvalContextExt<'
     /// This function will handle `goto_block` if needed.
```

Comment:
Looks like this comment no longer applies?

Reply:
I'm not sure what you mean - it still calls `goto_block` at the end of the function call, for foreign functions which need it.

Reply:
I thought something changed here because you removed the comment above saying "`goto_block` already handled".